tests/interpretation/test_rankings.py

```diff
@@ -0,0 +1,45 @@
+import random 
+import unittest 
+
+import neurox.interpretation.probeless as probeless 
+import neurox.interpretation.iou_probe as iou_probe
+import neurox.interpretation.gaussian_probe as guassian_probe
+import neurox.interpretation.linear_probe as linear_probe
+import numpy as np
+import torch 
+
+seed_value = 99
+np.random.seed(seed_value)
```

Comment:
We should set a few other seeds, `torch`'s/python's `random` to ensure the same env everytime